zombie_dice/ai.go

```diff
@@ -0,0 +1,24 @@
+package zombie_dice
+
+import (
+	"github.com/Akavall/GoGamesProject/dice"
+)
+
+func GreedyAI(shots int) int {
```

Comment:
Maybe. I don't know enough about interfaces to know if it is a good idea, this looks like a good place to practice :). But I know that AI's would require different inputs, Random requires no inputs a smarter AI would want to have all the information. 

What benefits would interface bring?